src/H5trace.c

```diff
@@ -156,11 +160,10 @@ H5_trace(const double *returning, const char *func, const char *type, ...)
         H5_timer_init(&running_timer);
```

Comment:
So, in some condition, the timer is initialized again?  Is that ok?

Reply:
H5_timer_init() just resets the timer fields.

Reply:
This one tripped me up when I first read it. Note that there's a `running_time` and a `function_timer`